<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLSliceViewWidget</class>
 <widget class="qMRMLWidget" name="qMRMLSliceViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qMRMLSliceViewWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="PushButtonSliceViewName">
     <property name="text">
      <string>RED</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qMRMLWidgets.qrc">
          <normaloff>:/Icons/SliceViewerOR.png</normaloff>:/Icons/SliceViewerOR.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="OrientationSelector">
        <property name="toolTip">
         <string>Select orientation</string>
        </property>
        <item>
         <property name="text">
          <string>Axial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sagittal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coronal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reformat</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="ForgroundLayerOptionButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qMRMLWidgets.qrc">
          <normaloff>:/Icons/SliceViewerFG.png</normaloff>:/Icons/SliceViewerFG.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="qMRMLNodeComboBox" name="ForegroundLayerNodeSelector">
        <property name="toolTip">
         <string>Select the foreground</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLVolumeNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="LabelMapLayerOptionButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qMRMLWidgets.qrc">
          <normaloff>:/Icons/SliceViewerLB.png</normaloff>:/Icons/SliceViewerLB.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="LabelMapSelector">
        <property name="toolTip">
         <string>Select the label map</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="BackgroundLayerOptionButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qMRMLWidgets.qrc">
          <normaloff>:/Icons/SliceViewerBG.png</normaloff>:/Icons/SliceViewerBG.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="qMRMLNodeComboBox" name="BackgroundLayerNodeSelector">
        <property name="toolTip">
         <string>Select the background</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLVolumeNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="LayerLinkToggle">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/LinkOff.png</normaloff>
            <normalon>:/Icons/LinkOn.png</normalon>:/Icons/LinkOff.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="LayerVisibilityToggle">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/VisibleOff.png</normaloff>
            <normalon>:/Icons/VisibleOn.png</normalon>:/Icons/VisibleOff.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="SliceMoreOptionButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/SliceMoreOptions.png</normaloff>:/Icons/SliceMoreOptions.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ctkSliderSpinBoxWidget" name="SliceOffsetSlider"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="ctkVTKSliceView" name="VTKSliceView">
        <property name="renderEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkVTKSliceView</class>
   <extends>QWidget</extends>
   <header>ctkVTKSliceView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderSpinBoxWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderSpinBoxWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qMRMLSliceViewWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>BackgroundLayerNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSliceViewWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>LabelMapSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLSliceViewWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ForegroundLayerNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
